config/config.exs

```diff
@@ -25,10 +25,10 @@ config :hexpm, Hexpm.Web.Endpoint,
 
 config :hexpm, Hexpm.RepoBase,
   priv: "priv/repo",
-  pool: DBConnection.Sojourn,
   protector: false,
   overload_alarm: false,
-  underload_alarm: false
+  underload_alarm: false,
```

Comment:
Do we still have the `protector` and `alarm` options when not using sbroker?

Reply:
I don't think so, but I might be wrong.

https://hexdocs.pm/db_connection/2.0.0-rc.0/DBConnection.ConnectionPool.html#module-options

Reply:
You're right, it was a sbroker option.